### DISCUSSION TITLE
Add system monitor to the bottom bar

### DISCUSSION THREAD

**Discussion by peq42** [linuxmint/discussions#640] [Ideas]
It'd be cool to have Linux Mint "borrow" windows' task bat option to Right click > Open System Monitor. A lot of non tech users coming from windows kinda expect that feature there.

Alternatively, having CTRL + Alt + Del(ete) pull it up by default would also be interesting.

**Comment by rcalixte** [comment]
> It'd be cool to have Linux Mint "borrow" windows' task bat option to Right click > Open System Monitor. A lot of non tech users coming from windows kinda expect that feature there.

As a workaround, you can add this menu action to the desktop using [this Action](https://cinnamon-spices.linuxmint.com/actions/view/6). It's not the same as the panel but hopefully it is close enough to get the desired effect.



> Alternatively, having CTRL + Alt + Del(ete) pull it up by default would also be interesting.

It may not be set by default, but you can configure this as a custom binding action in the Keyboard settings.

**Comment by peq42** [comment]
well yes I know these, but as mentioned, having it as default/on the bottom bar like that would be a lot nicer for new users/old folk that dont know how to use their PCs very well

**Reply by Jopp-gh** [reply]
Ex MacOs user here. Adding such a feature to defaults sounds a bit superfluous, as *Linux/Unix systems don't need a lot of superfluous stuff that comes by default with windows*.  
Users from different platforms love Linux Mint, because you can customize all by yourself. Forcing windows defaults on Linux isn't needed.

Still, I agree sometimes it's nice to monitor process activity, so you could do this:
- add a shortcut (as already mentioned above) 
-  add system monitor to your panel
- Alternatively, there should be an applet for this feature req. if I remember well

After half a year with Mint, I found out that Mint is more awesome than Mac. Only at the beginning we are unsettled how different Mint is.

**Reply by VewDev** [reply]
In my opinion, new users and older folks that don't know hot to use their PCs, won't know how to use the System Monitor either.

System Monitor is easily accessible through the apps list in the menu, so a very low level of skill is required to open it anyway.

A lot of the non-IT skilled Windows users I have dealt with as an IT person didn't know what the Task Manager (or any other administrative tool) was anyway.
